### ISSUE TITLE
NOT FOUND SCREEN

### ISSUE THREAD

**Issue by nano2sillery** [thijsbekke/Octoprint-LaMetric#7]
Thanks for this cooool plugin....i work with octoprint and my LAMETRIC screen display before printing Bed and HOTEND temperature.... and NOT FOUND Screen...... when printing is in progress, LAMETRIC display many NOT FOUND screen and some times Print Progression % screen. Could you help me ? Thks

**Comment by thijsbekke** [comment]
Hi, 

There are two different apps/plugins with different purposes. 

The purpose of the OctoPrint app on your LaMetric is too display the number of installs of a plugin, (you probally don't need this)  In the settings of this app (in the LaMetric app) you can enter your a OctoPrint plugin name. If none is entered "X Not Found" is displayed. 

Too display the progress of a print printed with OctoPrint no app on your LaMetric is required. 

Hope this helps


TLDR: Remove the app from your LaMetric, and you don't see the "Not Found" message.

Below a screenshot of the settings in de LaMetric app.
![Screenshot](https://user-images.githubusercontent.com/1152678/101793551-3762da80-3b06-11eb-9248-0957900b9b0d.png)
